easybuild/easyconfigs/h/HMMER/HMMER-3.1b2-foss-2016a.eb

```diff
@@ -28,16 +28,16 @@ description = """HMMER is used for searching sequence databases for homologs of
 toolchain = {'name': 'foss', 'version': '2016a'}
 
 source_urls = ['http://eddylab.org/software/hmmer%(version_major)s/%(version)s/']
-sources = ['hmmer-%(version)s-linux-intel-x86_64.tar.gz']
+sources = ['hmmer-%(version)s.tar.gz']
+
+runtest = 'check'
 
```

Comment:
Is this were `runtest` is expected to go? Just curious, as I would tend to put it later.